.travis.yml

```diff
@@ -34,9 +34,5 @@ before_script:
 script:
   - "make test"
   - "python manage.py makemessages -l eo"
-branches:
-    only:
-      - master
-      - ora-staff-grading
```

Comment:
Correct. See https://docs.travis-ci.com/user/customizing-the-build/#Building-Specific-Branches.